Feature/coherence 12.2.1 1 0

almubinsultan opened the pull request.

rstafford commented:
Thank you Mohammad.

rstafford commented:
Hi Mohammad,

Could you please sign the Oracle Contributor Agreement?  See http://www.oracle.com/technetwork/community/oca-486395.html

Cheers,
Randy

> On Oct 18, 2016, at 3:15 AM, Mohammad Al-Mubin Sultan <notifications@github.com> wrote:
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/coherence-community/coherence-hibernate-integration/pull/1 <https://github.com/coherence-community/coherence-hibernate-integration/pull/1>
> Commit Summary
> 
> Updated .gitignore file for ignoring .ipr and .iws files
> Added profile for Coherence 12.2.1-1-0
> Made 12.2.1-1-0 the default Coherence version
> File Changes
> 
> M .gitignore <https://github.com/coherence-community/coherence-hibernate-integration/pull/1/files#diff-0> (2)
> M pom.xml <https://github.com/coherence-community/coherence-hibernate-integration/pull/1/files#diff-1> (8)
> Patch Links:
> 
> https://github.com/coherence-community/coherence-hibernate-integration/pull/1.patch <https://github.com/coherence-community/coherence-hibernate-integration/pull/1.patch>
> https://github.com/coherence-community/coherence-hibernate-integration/pull/1.diff <https://github.com/coherence-community/coherence-hibernate-integration/pull/1.diff>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub <https://github.com/coherence-community/coherence-hibernate-integration/pull/1>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AD4XJXFFcfh72aJ2tpNFom-6hF5ZZI_Cks5q1I4mgaJpZM4KZja6>.
>

almubinsultan commented:
Randy,

Some background on the pull request. All code remains same as it is for
hibernate 4. Few changes that required to make it work for hibernate 3.

- Removed NaturalIdCache support related code as it does not exists in
hibernate 3
- Used hibernate 3 RegionFactory, basically had to change a whole bunch of
import to make it work in hibernate 3
- Created only one interface RegionAccessor, this was introduced by
hibernate in their version 4 but i liked the approach so created one for
hibernate 3 to keep the implementation code no changed from yours.
- Changed the AccessType enum comparison switch cases code to if/else as in
hibernate 3 the class is a concrete class.

Other than this all other code is same as yours.

If you want i can send you a diff to make it easy for you to review. Please
let me know.

Hope this helps in reviewing.



On Wed, Oct 26, 2016 at 3:54 AM, rstafford <notifications@github.com> wrote:

> I’ve seen in other threads that you’ve signed the Oracle Contributor
> Agreement, and created a pull request for the work you did. Thank you for
> contributing to the project. Let me review your pull request at my earliest
> opportunity.
>
> Cheers,
> Randy
>
>
> > On Oct 25, 2016, at 12:45 PM, Mohammad Al-Mubin Sultan <
> notifications@github.com> wrote:
> >
> > Hi,
> >
> > I am using coherence-12.x + hibernate 3 + weblogic-12.x, The
> > coherence-hibernate-integration project is based on hibernate 4. I am
> > working on getting it running for hibernate 3. I do also want to
> contribute
> > back the source to the community as i am working based on the core
> project
> > that is currently on hibernate 4.
> >
> > Please let me know how should i proceed.
> >
> > On Tue, Oct 18, 2016 at 8:46 PM, rstafford <notifications@github.com>
> wrote:
> >
> > > Hi Mohammad,
> > >
> > > Could you please sign the Oracle Contributor Agreement? See
> > > http://www.oracle.com/technetwork/community/oca-486395.html
> > >
> > > Cheers,
> > > Randy
> > >
> > > > On Oct 18, 2016, at 3:15 AM, Mohammad Al-Mubin Sultan <
> > > notifications@github.com> wrote:
> > > >
> > > > You can view, comment on, or merge this pull request online at:
> > > >
> > > > https://github.com/coherence-community/coherence-hibernate-
> > > integration/pull/1 <https://github.com/coherence-
> > > community/coherence-hibernate-integration/pull/1>
> > > > Commit Summary
> > > >
> > > > Updated .gitignore file for ignoring .ipr and .iws files
> > > > Added profile for Coherence 12.2.1-1-0
> > > > Made 12.2.1-1-0 the default Coherence version
> > > > File Changes
> > > >
> > > > M .gitignore <https://github.com/coherence-
> > > community/coherence-hibernate-integration/pull/1/files#diff-0> (2)
> > > > M pom.xml <https://github.com/coherence-
> community/coherence-hibernate-
> > > integration/pull/1/files#diff-1> (8)
> > > > Patch Links:
> > > >
> > > > https://github.com/coherence-community/coherence-hibernate-
> > > integration/pull/1.patch <https://github.com/coherence-
> > > community/coherence-hibernate-integration/pull/1.patch>
> > > > https://github.com/coherence-community/coherence-hibernate-
> > > integration/pull/1.diff <https://github.com/coherence-
> > > community/coherence-hibernate-integration/pull/1.diff>
> > > > —
> > > > You are receiving this because you are subscribed to this thread.
> > > > Reply to this email directly, view it on GitHub <
> > > https://github.com/coherence-community/coherence-hibernate-
> > > integration/pull/1>, or mute the thread <https://github.com/
> > > notifications/unsubscribe-auth/AD4XJXFFcfh72aJ2tpNFom-
> > > 6hF5ZZI_Cks5q1I4mgaJpZM4KZja6>.
> > >
> > > >
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/coherence-community/coherence-hibernate-
> integration/pull/1#issuecomment-254530898>,
> > > or mute the thread
> > > <https://github.com/notifications/unsubscribe-auth/
> AFD1kuAuLaAlEpobzdaxNxpy5PvbOEbHks5q1NvagaJpZM4KZja6>
> > > .
> > >
> >
> >
> >
> > --
> > Thank You
> > Mohammad Al- Mubin Sultan
> > Software Engineer
> > +8801611081429
> > +8801712076039
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub <
> https://github.com/coherence-community/coherence-hibernate-
> integration/pull/1#issuecomment-256135546>, or mute the thread <
> https://github.com/notifications/unsubscribe-auth/AD4XJdBzGEbI_
> zqbO3s4LPQAnO1NFeuwks5q3k5PgaJpZM4KZja6>.
>
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coherence-community/coherence-hibernate-integration/pull/1#issuecomment-256187425>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFD1kikLVVvLF8-XyMHmME3jvXQGeaNoks5q3nqIgaJpZM4KZja6>
> .
>



-- 
            Thank You
Mohammad Al- Mubin Sultan
Software Engineer
+8801611081429
+8801712076039

rstafford commented:
Thanks Mubin.  I don’t expect to have any technical objection; it’s just a matter of time management and work prioritization.

Cheers,
Randy

> On Oct 27, 2016, at 9:40 AM, Mohammad Al-Mubin Sultan <notifications@github.com> wrote:
> 
> Randy,
> 
> Some background on the pull request. All code remains same as it is for
> hibernate 4. Few changes that required to make it work for hibernate 3.
> 
> - Removed NaturalIdCache support related code as it does not exists in
> hibernate 3
> - Used hibernate 3 RegionFactory, basically had to change a whole bunch of
> import to make it work in hibernate 3
> - Created only one interface RegionAccessor, this was introduced by
> hibernate in their version 4 but i liked the approach so created one for
> hibernate 3 to keep the implementation code no changed from yours.
> - Changed the AccessType enum comparison switch cases code to if/else as in
> hibernate 3 the class is a concrete class.
> 
> Other than this all other code is same as yours.
> 
> If you want i can send you a diff to make it easy for you to review. Please
> let me know.
> 
> Hope this helps in reviewing.
> 
> 
> 
> On Wed, Oct 26, 2016 at 3:54 AM, rstafford <notifications@github.com> wrote:
> 
> > I’ve seen in other threads that you’ve signed the Oracle Contributor
> > Agreement, and created a pull request for the work you did. Thank you for
> > contributing to the project. Let me review your pull request at my earliest
> > opportunity.
> >
> > Cheers,
> > Randy
> >
> >
> > > On Oct 25, 2016, at 12:45 PM, Mohammad Al-Mubin Sultan <
> > notifications@github.com> wrote:
> > >
> > > Hi,
> > >
> > > I am using coherence-12.x + hibernate 3 + weblogic-12.x, The
> > > coherence-hibernate-integration project is based on hibernate 4. I am
> > > working on getting it running for hibernate 3. I do also want to
> > contribute
> > > back the source to the community as i am working based on the core
> > project
> > > that is currently on hibernate 4.
> > >
> > > Please let me know how should i proceed.
> > >
> > > On Tue, Oct 18, 2016 at 8:46 PM, rstafford <notifications@github.com>
> > wrote:
> > >
> > > > Hi Mohammad,
> > > >
> > > > Could you please sign the Oracle Contributor Agreement? See
> > > > http://www.oracle.com/technetwork/community/oca-486395.html
> > > >
> > > > Cheers,
> > > > Randy
> > > >
> > > > > On Oct 18, 2016, at 3:15 AM, Mohammad Al-Mubin Sultan <
> > > > notifications@github.com> wrote:
> > > > >
> > > > > You can view, comment on, or merge this pull request online at:
> > > > >
> > > > > https://github.com/coherence-community/coherence-hibernate-
> > > > integration/pull/1 <https://github.com/coherence-
> > > > community/coherence-hibernate-integration/pull/1>
> > > > > Commit Summary
> > > > >
> > > > > Updated .gitignore file for ignoring .ipr and .iws files
> > > > > Added profile for Coherence 12.2.1-1-0
> > > > > Made 12.2.1-1-0 the default Coherence version
> > > > > File Changes
> > > > >
> > > > > M .gitignore <https://github.com/coherence-
> > > > community/coherence-hibernate-integration/pull/1/files#diff-0> (2)
> > > > > M pom.xml <https://github.com/coherence-
> > community/coherence-hibernate-
> > > > integration/pull/1/files#diff-1> (8)
> > > > > Patch Links:
> > > > >
> > > > > https://github.com/coherence-community/coherence-hibernate-
> > > > integration/pull/1.patch <https://github.com/coherence-
> > > > community/coherence-hibernate-integration/pull/1.patch>
> > > > > https://github.com/coherence-community/coherence-hibernate-
> > > > integration/pull/1.diff <https://github.com/coherence-
> > > > community/coherence-hibernate-integration/pull/1.diff>
> > > > > —
> > > > > You are receiving this because you are subscribed to this thread.
> > > > > Reply to this email directly, view it on GitHub <
> > > > https://github.com/coherence-community/coherence-hibernate-
> > > > integration/pull/1>, or mute the thread <https://github.com/
> > > > notifications/unsubscribe-auth/AD4XJXFFcfh72aJ2tpNFom-
> > > > 6hF5ZZI_Cks5q1I4mgaJpZM4KZja6>.
> > > >
> > > > >
> > > >
> > > > —
> > > > You are receiving this because you authored the thread.
> > > > Reply to this email directly, view it on GitHub
> > > > <https://github.com/coherence-community/coherence-hibernate-
> > integration/pull/1#issuecomment-254530898>,
> > > > or mute the thread
> > > > <https://github.com/notifications/unsubscribe-auth/
> > AFD1kuAuLaAlEpobzdaxNxpy5PvbOEbHks5q1NvagaJpZM4KZja6>
> > > > .
> > > >
> > >
> > >
> > >
> > > --
> > > Thank You
> > > Mohammad Al- Mubin Sultan
> > > Software Engineer
> > > +8801611081429
> > > +8801712076039
> > > —
> > > You are receiving this because you modified the open/close state.
> > > Reply to this email directly, view it on GitHub <
> > https://github.com/coherence-community/coherence-hibernate-
> > integration/pull/1#issuecomment-256135546>, or mute the thread <
> > https://github.com/notifications/unsubscribe-auth/AD4XJdBzGEbI_
> > zqbO3s4LPQAnO1NFeuwks5q3k5PgaJpZM4KZja6>.
> >
> > >
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/coherence-community/coherence-hibernate-integration/pull/1#issuecomment-256187425>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/AFD1kikLVVvLF8-XyMHmME3jvXQGeaNoks5q3nqIgaJpZM4KZja6>
> > .
> >
> 
> 
> 
> -- 
> Thank You
> Mohammad Al- Mubin Sultan
> Software Engineer
> +8801611081429
> +8801712076039
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/coherence-community/coherence-hibernate-integration/pull/1#issuecomment-256680499>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AD4XJfnHeokGAE-Wur3ftxDT8WdNvz-9ks5q4MX7gaJpZM4KZja6>.
>

almubinsultan commented:
Dear Concern,

Any update on this PR? Just following up. Have a great day.

Thanks
Mubin
